### ISSUE TITLE
Run task if file does not exist

### ISSUE THREAD

**Issue by roblabla** [sagiegurari/cargo-make#287]
**Features Description**
It'd be nice to be able to make a task only run if a file does not exist. For instance, I have a task that generates some development signing keys. I only need it to run if the key is not already generated. I currently use an `if [ -f ]` in a condition_script, but that's not portable (it won't work on windows).

**Describe the solution you'd like**
I'd like to extend `condition`'s criterias to add a `file_not_exists` key. The value would be an array of paths. If **any** of the listed path don't exist, the condition is met and the task is run.

The any part runs a bit counter to the rest of the conditions (which all requires that **all** requirements are met), but I'm not sure of any better way to express my requirements.

Alternatively, maybe we could make shell2batch convert `if [ -f $FILE]`, but that seems like it might be very hard to get right.

**Code Sample**
```toml
[tasks.gen_secureboot_keys]
condition = { "file_not_exists" = ["external/PK.key", "external/PK.crt"] }
script = ['''
# Generate keys
touch external/PK.key external/PK.crt
''']
```

**Comment by sagiegurari** [comment]
a condition is a good idea.
if i'm implementing not exists, i'll implement exists as well.
i'll probably be able to start working on next version in about a week or so...

**Comment by sagiegurari** [comment]
@roblabla this is now implemented in 0.22.2 branch. see [docs](https://github.com/sagiegurari/cargo-make/tree/0.22.2#usage-conditions-structure) for files_exist and files_not_exist conditions.

**Comment by sagiegurari** [comment]
published in 0.22.2 release
